pkg/evaluator/http.go

```diff
@@ -0,0 +1,128 @@
+package evaluator
```

Comment:
Nice work! I think that you just extended openmock to the unittest space. With evaluator package. You can define a remote http/grpc/kafka/etc in yaml and use evaluator for some simple unit tests in go.